frontend/app/model/Eventbrite.scala

```diff
@@ -86,6 +86,11 @@ object Eventbrite {
 
   case class EBPricing(value: Int) extends EBObject {
     lazy val formattedPrice = formatPriceWithCurrency(value)
+    def addFee(other: Option[EBPricing]) : Option[EBPricing] = {
```

Comment:
Why on earth would I want to add an "optional" fee? Wouldn't be better something like?

`addFee(that: EBPricing): EBPricing`